include/services/error_indexes.h

```diff
@@ -145,7 +145,7 @@ enum ErrorID
     ErrorBufferSizeIntegerOverflow = -80,                               /*!< Integer oveflow is occured during buffer size calculation */
 
     // Environment errors: -2000..-2999
-    ErrorCpuIsInvalid = -1,                                             /*!< Invalid CPU value used */
+    ErrorCpuIsInvalid = -2999,                                          /*!< Invalid CPU value used */
```

Comment:
Looks like it breaks backward compatibility

Reply:
@ntfshard do you plan fix the backward compatibility problem or close the PR?